src/main/java/frc/robot/subsystems/kitbot/KitbotRoller.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.kitbot;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkMaxConfig;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+
+public class KitbotRoller extends SubsystemBase {
+  public static final int ROLLER_MOTOR_ID = 5;
```

Comment:
This is a duplicate. Use the one in Constants. CAN IDs need to be unique, so they belong in the global Constants file